lambda-function/src/utils/measureConversionUtils.ts

```diff
@@ -87,12 +103,48 @@ const convertMeasureGroups = (measureResourceJson: string): Array<Group> => {
       scoring: measureResource.scoring.coding[0].display,
     } as Group;
 
-    const populations = convertPopulations(group.population);
-    madieMeasureGroup.population = populations;
+    //const population = convertPopulations(group.population);
+    const populations: Population[] = [];
+    let result = Object.entries(group.population);
+    result.map((item) => {
```

Comment:
if you don't want to return anything (seems like mapping is not happening) either you can use foreach/for of or modify mapping like:

```
populations = result.map((item) => {
      return convertPopulations(item[1]);
    }); 
```

Reply:
@adongare Ok did some work on this, just checked the code in, thanks!